hack/install-operator.sh

```diff
@@ -6,24 +6,46 @@ BASEPATH=$( dirname $SELF )
 
 NAMESPACE=${1:-kubevirt}
 
+_curl() {
+	# this dupes the baseline "curl" command line, but is simpler
+	# wrt shell quoting/expansion.
+	if [ -n "${GITHUB_TOKEN}" ]; then
+		curl -H "Authorization: token ${GITHUB_TOKEN}" $@
+	else
+		curl $@
+	fi
+}
+
 oc create -f ${BASEPATH}/../deploy/crds/kubevirt_v1_commontemplatesbundle_crd.yaml
 oc create -f ${BASEPATH}/../deploy/crds/kubevirt_v1_nodelabellerbundle_crd.yaml
 oc create -f ${BASEPATH}/../deploy/crds/kubevirt_v1_templatevalidator_crd.yaml
 
+LAST_TAG=""
+if [ "${CI}" != "true" ] || [ "${TRAVIS}" != "true" ]; then
+	# TODO: consume releases, not tags
+	LAST_TAG=$( _curl -s https://api.github.com/repos/MarSik/kubevirt-ssp-operator/tags | jq -r '.[0].name' )
```

Comment:
are these releases sorted from newest to oldest? Because e.g. you create release 1.2.0 and then you realize in 1.1.0 was mistake, so you create 1.1.1 release and now newest release is 1.1.1 not 1.2.0

Reply:
Good point. I should check and make sure to pick the latest.

Reply:
This might solve it:
`curl -s https://api.github.com/repos/MarSik/kubevirt-ssp-operator/releases|grep tag_name|sort -V | tail -1 | awk -F':' '{print $2}' | sed 's/,//' | xargs`
(taken from https://kubevirt.io/quickstart_minikube/)